adr/recipes/push_failure.py

```diff
@@ -0,0 +1,16 @@
+"""
+Get information on the ActiveData schema. The command returns the
```

Comment:
I think this first sentence might be left over from the previous recipe you copied from.